sandbox/urls.py

```diff
@@ -1,10 +1,7 @@
 from django.conf.urls import patterns, include, url
 from django.contrib import admin
 
-urlpatterns = patterns('',
-    # Examples:
-    # url(r'^$', 'sandbox.views.home', name='home'),
-    # url(r'^blog/', include('blog.urls')),
-
+urlpatterns = patterns(
+    '',
```

Comment:
Unnecessary change. This is just the default file created by Django's startproject and it's not used otherwise.